nix/overlay.nix

```diff
@@ -54,7 +54,7 @@ let
             (
               self: super:
                 {
-                  openapi3-code-generator = buildStrictly (final.haskellPackages.callCabal2nixWithOptions "openapi3-code-generator" (final.gitignoreSource ../openapi3-code-generator) "--no-hpack" { });
+                  openapi3-code-generator = buildStrictly (final.haskellPackages.callCabal2nixWithOptions "openapi3-code-generator" ../openapi3-code-generator "--no-hpack" { });
```

Comment:
Is this change flakes related or is it just a general optimization because cabal will only regard files listed in the cabal file anyways?